spec/models/subscription_spec.rb

```diff
@@ -25,7 +25,18 @@
     it { is_expected.to validate_presence_of(:plan_id) }
     it { is_expected.to validate_presence_of(:user_id) }
     it { is_expected.to validate_presence_of(:project_id) }
-    it { is_expected.to validate_presence_of(:donator_cpf).on(:create) }
+
+    context "when the locale is Portuguese" do
+      before { I18n.locale = :pt }
```

Comment:
You could have two validations here for when the locale is `:pt`, one that fails and one that passes.